tools/versioner/cmd/root_test.go

```diff
@@ -344,6 +365,9 @@ go 1.12
 	if !strings.EqualFold(string(b), after) {
 		t.Fatalf("bad go.mod file, expected '%s' got '%s'", after, string(b))
 	}
+	if !fileExists(filepath.Join("../testdata/scenariob/foo/v2", "CHANGELOG.md")) {
+		t.Fatal("expected changelog in scenariob")
```

Comment:
>scenariob [](start = 33, length = 9)

hmm what is scenariob ? would it be possible to give them more semanticNames ?

Reply:
I added comment text in each foo.go file that describes the scenario, figured that might be better than trying to get create with the names.  I will double-check that I didn't miss one.